generators/app/templates/src/config/reactotronConfig.ejs

```diff
@@ -3,45 +3,41 @@ import Reactotron, { asyncStorage, overlay, trackGlobalErrors } from 'reactotron
 import apisaucePlugin from 'reactotron-apisauce';
 import { reactotronRedux } from 'reactotron-redux';
 import { NativeModules } from 'react-native';
+import { Tron } from '@interfaces/reactotron';
+// Console augmentation
+declare global {
+  interface Console {
+    tron: Tron;
+  }
+}
 
 // If you want to use a physical device and connect it to reactotron, execute first `adb reverse tcp:9090 tcp:9090`
 if (__DEV__) {
   const { scriptURL } = NativeModules.SourceCode;
   const scriptHostname = scriptURL.split('://')[1].split(':')[0];
   Reactotron.configure({ name: '<%= projectName %>', host: scriptHostname })
-    .use(trackGlobalErrors())
+    .use(trackGlobalErrors({}))
     .use(apisaucePlugin())
     .use(
       reactotronRedux({
         onRestore: state =>
           Object.entries(state).reduce(
+            // eslint-disable-next-line new-cap
```

Comment:
I think this  `eslint-disable` is not necessary. The rule is already deactivated in the [.eslintrc](https://github.com/Wolox/wolmo-bootstrap-react-native/blob/master/generators/app/templates/.eslintrc.ejs) file